spec/models/article_spec.rb

```diff
@@ -16,19 +16,20 @@
     expect(article.valid?).to be(false)
   end
 
-  it 'will not createa an article with a duplicate title' do
+  it 'will not create an article with a duplicate title' do
     @article = described_class.create({ title: 'First!',
```

Comment:
I dont believe this needs to be an instance variable or even a reg variable. All you're doing is creating an article in the database and then attempting to create another article of the same title.

Reply:
It's also incongruent with the rest of Gloria's test. Wonder if it was just an oversight.